### PR TITLE
Create moveit_commander.rst

### PR DESCRIPTION
**Pull request by 0ldham**
Another tutorial from @davetcoleman's old MoveIt wiki.  Won't work without [PR #581](https://github.com/ros-planning/moveit/pull/581).

### CODE REVIEW

**Comment by davetcoleman** [comment]
Thanks @0ldham!

for the tutorials you are adding can you also add them to the [home page index](https://github.com/ros-planning/moveit_tutorials/blob/kinetic-devel/index.rst) so they are easier to find?

File: doc/pr2_tutorials/planning/src/doc/moveit_commander.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+=======================================
+
+The `moveit_commander <http://wiki.ros.org/moveit_commander>`_ Python package offers wrappers for the functionality provided in MoveIt!. Simple interfaces are available for motion planning, computation of Cartesian paths, and pick and place.
+
+Command Line Tool
+-----------------
+
+The **moveit_commander** package also includes a command line tool for controlling the robot. 
+
+First, launch the MoveIt! demo interface: ::
```
Comment (v4hn) [inline doc/pr2_tutorials/planning/src/doc/moveit_commander.rst]: This is only needed if you did not start a `move_group` node with your real robot,
but you can (of course) run the MoveIt commander with the real robot too.

Could you please clarify here that the "demo" mode is the simulated moveit environment?

File: doc/pr2_tutorials/planning/src/doc/moveit_commander.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+This will connect to the move_group node for the group name you specified (in the PR2, for instance, you could connect to left_arm). You can now execute commands on that group. 
+This command, **current**, will show you the current state of your group: ::
+
+ current
+
+To record that state under a specific name you can simply type: ::
+
+ rec c
+ 
+This will remember the current joint values of the robot group under the name c. To get the robot to move, you could type, for example: ::
```
Comment (v4hn) [inline doc/pr2_tutorials/planning/src/doc/moveit_commander.rst]: Could you move the description of the following code up here?
I know way too many people who would copy&paste the code before reading on, possibly with there own real robot connected... :/

**Comment by v4hn** [comment]
ping @0ldham ?

**Comment by 0ldham** [comment]
@v4hn sorry for not getting to this sooner! I have updated the code accordingly.

File: doc/pr2_tutorials/planning/src/doc/moveit_commander.rst [hunk 3]
```diff
@@ -29,13 +29,14 @@ To record that state under a specific name you can simply type: ::
 
  rec c
  
-This will remember the current joint values of the robot group under the name c. To get the robot to move, you could type, for example: :: 
+This will remember the current joint values of the robot group under the name c. Matlab-like syntax is available for modifying joint values. The code above copies the joint values of c into a new variable named goal. We then modify the first joint of goal to 0.2. You may need to use a different value instead of 0.2 (it needs to be within your allowed bounds and not cause collisions). The **go** command plans a motion and executes it.
```
Comment (v4hn) [inline doc/pr2_tutorials/planning/src/doc/moveit_commander.rst]: "The code above" should be changed to "The code below" now that you moved the block.

**Comment by davetcoleman** [comment]
I have fixed the change requested by @v4hn, merging.

**Pull request merged**